src/XML/src/FMI3/fmi3_xml_model_structure.c

```diff
@@ -94,7 +94,7 @@ jm_vector(jm_voidp)* fmi3_xml_get_event_indicators(fmi3_xml_model_structure_t* m
 
 static int fmi3_xml_get_dependencies(jm_vector(jm_voidp)* msVector, 
         fmi3_xml_dependencies_t* dep, fmi3_xml_variable_t* variable,
-        size_t* numDependencies, size_t** dependency, char** factorKind) 
+        size_t* numDependencies, bool* dependsOnAll, size_t** dependencies, char** dependenciesKind)
```

Comment:
To keep things consistent with the rest of the API, I think we should make it `int* dependsOnAll`. (Unfortunately we do on some places use 'char' and 'unsigned int', so it would still not be fully consistent, but one less at least.) Possibly we could change all the 3.0 API to use bool in another ticket. Should not be too much work.

Reply:
Changed to int.